keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1008,12 +1052,19 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+kube-apiserver must support it, and every node that has a pod scheduled that
+attempts to mount image volumes must have a kubelet and CRI that support it.
+If any don't, the volume won't be enabled and containers will not start. If the
+kubelet does not support it but will be upgraded, then the container will become
+available after the upgrade.
+
+Since this field only applies on a per node basis, once the control plane agrees
```

Comment:
We have a possible of component feature gate skew here (ie kube-apiserver has feature gate off and kubelet has feature on). What is the expected behavior for this? And if we flip it (kubelet off and kube-apiserver on)?

Reply:
If the API is not available then the feature cannot be used by the nodes because the fields are not set. 

If the API is available but the kubelet has not enabled it, then the fields will be ignored by the kubelet.

Reply:
That is pretty clear. Maybe update the text to state that.

Reply:
> If the API is available but the kubelet has not enabled it, then the fields will be ignored by the kubelet.

We've been around a couple times, I thought @haircommander told me that old kubelets rejected usernamespace pods.